docs/UserGuide.adoc

```diff
@@ -341,176 +343,193 @@ otherwise the application will hang.
+otherwise the application will hang.
 
 *Examples:*
 
-* `reopenissue 1`
-* `reopenissue 3`
+* `complete 3`
+* `ct 3`
 
-=== Resetting a task as incompleted: `reset` OR `rt`
+==== Resetting a task as incompleted: `reset` OR `rt`
 
-Reset a task as incompleted. +
+You can reset a task as incompleted when you want to work on it. +
 *Format:* `reset INDEX` OR `rt INDEX`
 
 ****
-*Instructions:*
+*Notes:*
 
-* Reset the task with index number `INDEX` as incompleted.
+* You can reset the task with index number `INDEX` as incompleted.
 * The index refers to the index number shown in the most recent listing.
 * The index *must be a positive integer* 1, 2, 3, ...
 ****
 
 [TIP]
-The command requires Internet connection. You may be brought to a login page in your browser. Please do not close the tab without accepting/declining request,
-otherwise the application will hang.
+An issue has only 'title' field as compulsory. Rest are all optional. +
+An issue can have more than one assignees and labels.
 
 *Examples:*
 
-* `reset 3`
-* `rt 3`
+* `+issue ti/complete issue a/johndoe ms/v1.1 b/CS2103T is a software engineering module l/type.task l/CS2103T`
+* `+issue ti/TestIssue`
+* `ci ti/Issue with only body b/test body l/type.test`
 
-=== Selecting a teammate : `select` OR `s`
+=== Teammate
 
-Selects the teammate identified by the index number used in the last teammate listing. +
-*Format:* `select INDEX` OR `s INDEX`
+==== Adding a teammate: `add` OR `a`
 
-****
-*Instructions:*
+You can add the contact information of a new teammate to the ProgressChecker. +
+*Format:* `add n/NAME p/PHONE_NUMBER e/EMAIL m/MAJOR y/YEAR [t/TAG]...` +
+OR +
+`a n/NAME p/PHONE_NUMBER e/EMAIL m/MAJOR y/YEAR [t/TAG]...`
 
-* Selects the teammate and loads the Google search page the teammate at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* `1, 2, 3, ...`
-****
+[TIP]
+A teammate can have any number of tags (including 0)
 
 *Examples:*
 
-* `list` +
-`select 2` +
-Selects the 2nd teammate in the ProgressChecker.
-* `find Betsy` +
-`s 1` +
-Selects the 1st teammate in the results of the `find` command.
+* `add n/John Doe p/98765432 e/johnd@example.com m/Computer Science y/2`
+* `a n/John Doe p/98765432 e/johnd@example.com m/Computer Science y/2`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com m/Computer Engineering p/1234567 y/3 t/criminal`
+* `a n/Betsy Crowe t/friend e/betsycrowe@example.com m/Information Security y/2 p/1234567 t/criminal`
 
-=== Sorting all teammates : `sort`
+==== Editing a teammate : `edit` OR `ed`
 
-Sorts all teammates in the ProgressChecker with their names in alphabetical order. +
-*Format:* `sort`
+You can edit the information of the certain existing teammate in the ProgressChecker. +
+*Format:* `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
+OR +
+`ed INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
```

Comment:
add these fields too: [m/MAJOR] [y/YEAR] [g/GITHUB_USERNAME].
also, in "add" command, it's stated as [p/PHONE_NUMBER], might want to make it consistent here?